README.md

```diff
@@ -10,8 +10,8 @@ This code is released under a GNU GPL v2 license (see LICENSE file for more info
 
 ## Usage
 
-1. Run the trajectory controller through baxter_trajectory_controller.launch. It seems that it has now been renamed to Joint Trajectory Action Server.
+Run tictactoe.launch
+
+	rosrun tictactoe tictactoe.launch
 
-2. Run tictactoe.launch
 
```

Comment:
Change the `README.md` so that it includes exactly the same things you wrote into the `package.xml` (so that any visitor of the repo can understand how to use it also within the repo).